service/arm_templates/security_groups_setup.json

```diff
@@ -212,7 +186,7 @@
               "protocol": "tcp",
               "sourcePortRange": "*",
               "destinationPortRange": "[string(parameters('kubernetesIngressInsecurePort'))]",
-              "sourceAddressPrefix": "[parameters('ingressLoadBalancerCidr')]",
+              "sourceAddressPrefix": "*",
```

Comment:
I think this rule is a dupe of previous one.

Reply:
This one is for the insecure port, the previous for the secure port.